### PR TITLE
[Shop][Behat] Add removing cart item feature

### PR DESCRIPTION
**Pull request by Arminek**
| Q               | A
| --------------- | ---
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | no
| Related tickets | #4939
| License         | MIT

### CODE REVIEW

File: features/cart/viewing_cart/removing_cart_item.feature [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        And I should be notified that my cart is empty
+
+    @todo
+    Scenario: Removing cart item from cart with multiple items
+        Given the cart also has "T-shirt potato"
+        And I open cart summary page
+        When I delete this product
+        Then I should be notified that this product has been successfully deleted
+        And unit price value should be "€12.54"
+        And I should see "Total"
```
Comment (pamil) [inline features/cart/viewing_cart/removing_cart_item.feature]: Do we need all those `And I should see "X" And X should be Y`? Isn't `And X should be Y` enough?

Comment (Arminek) [inline features/cart/viewing_cart/removing_cart_item.feature]: IMO we don't need those.

File: features/cart/viewing_cart/removing_cart_item.feature [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        And total value should be "€12.54"
+        And I should see "Qty"
+        And quantity value should be 1
+        And I should see "Shipping total"
```
Comment (pamil) [inline features/cart/viewing_cart/removing_cart_item.feature]: Is shipping relevant here?

File: features/cart/viewing_cart/removing_cart_item.feature [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    Scenario: Removing cart item from cart with multiple items
+        Given the cart also has "T-shirt potato"
+        And I open cart summary page
+        When I delete this product
+        Then I should be notified that this product has been successfully deleted
+        And unit price value should be "€12.54"
+        And I should see "Total"
+        And total value should be "€12.54"
+        And I should see "Qty"
+        And quantity value should be 1
```
Comment (pamil) [inline features/cart/viewing_cart/removing_cart_item.feature]: And there should be only one product in my cart?

File: features/cart/viewing_cart/removing_cart_item.feature [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+        Then I should be notified that this product has been successfully deleted
+        And I should be notified that my cart is empty
+
+    @todo
+    Scenario: Removing cart item from cart with multiple items
+        Given the cart also has "T-shirt potato"
+        And I open cart summary page
+        When I delete this product
+        Then I should be notified that this product has been successfully deleted
+        And unit price value should be "€12.54"
```
Comment (pamil) [inline features/cart/viewing_cart/removing_cart_item.feature]: ?

**Comment by Arminek** [comment]
In favor #4939

**Pull request closed without merging**